tools/walletextension/walletextension_container.go

```diff
@@ -32,8 +32,15 @@ func NewContainerFromConfig(config wecommon.Config, logger gethlog.Logger) *Cont
 	// create the account manager with a single unauthenticated connection
 	hostRPCBindAddrWS := wecommon.WSProtocol + config.NodeRPCWebsocketAddress
 	hostRPCBindAddrHTTP := wecommon.HTTPProtocol + config.NodeRPCHTTPAddress
+
+	// TODO fix passing this random key to next enclave and not generating it here every time we start the wallet extension
+	randomKey, err := wecommon.GenerateRandomKey()
```

Comment:
the randomkey needs to be persisted sealed by the current enclave, and read on restarts.
Have a look at: edglessdb.go:288

Reply:
this will be part of the next PR